src/test/java/ioananadroflorea/tests/Tema5/DemoQA.java

```diff
@@ -0,0 +1,111 @@
+package ioananadroflorea.tests.Tema5;
+
+import ioanaandroflorea.driver.BrowserManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.Keys;
+
+public class DemoQA {
+    public static void main(String[] args) {
+        textBox();
+        checkBox();
+        radioButton();
+        webTables();
+        button();
+    }
+
+    public static void textBox(){
+        ChromeDriver driver = BrowserManager.createChromeDriver();
+        driver.get("https://demoqa.com/elements");
+        WebElement textBox = driver.findElement(By.xpath("/html/body/div[2]/div/div/div[2]/div[1]/div/div/div[1]/div/ul/li[1]/span"));
+        textBox.click();
+        WebElement fullNameTextBox = driver.findElement(By.id("userName"));
+        fullNameTextBox.click();
+        fullNameTextBox.clear();
+        fullNameTextBox.sendKeys("John Doe");
+        WebElement emailTextBox = driver.findElement(By.id("userEmail"));
+        emailTextBox.click();
+        emailTextBox.clear();
+        emailTextBox.sendKeys("johndoe@yahoo.com");
+        WebElement submitButton = driver.findElement(By.id("submit"));
+        submitButton.sendKeys(Keys.ENTER);
+        driver.quit();
+    }
+
+    public static void checkBox(){
+        ChromeDriver driver = BrowserManager.createChromeDriver();
+        driver.get("https://demoqa.com/elements");
+        WebElement checkBox = driver.findElement(By.id("item-1"));
+        checkBox.click();
+        WebElement clickBoxHome = driver.findElement(By.cssSelector("span.rct-title"));
+        clickBoxHome.click();
+        WebElement textAfisat = driver.findElement(By.id("result"));
+        textAfisat.getText();
+        System.out.println(textAfisat.getText());
+        WebElement clickPlus = driver.findElement(By.cssSelector("#tree-node > div > button.rct-option.rct-option-expand-all"));
+        clickPlus.click();
+        WebElement clickMinus = driver.findElement(By.cssSelector("#tree-node > div > button.rct-option.rct-option-collapse-all"));
+        clickMinus.click();
+        driver.quit();
+    }
+    public static void radioButton(){
+        ChromeDriver driver = BrowserManager.createChromeDriver();
+        driver.get("https://demoqa.com/elements");
+        WebElement radioButton = driver.findElement(By.id("item-2"));
+        radioButton.click();
+        WebElement radioButtonAnswer = driver.findElement(By.id("impressiveRadio"));
```

Comment:
When clicking on radio buttons, try to use either JavascriptExecutor or use the label instead of input tag when searching the element. Now it fails at the click action ->
Exception in thread "main" org.openqa.selenium.ElementClickInterceptedException: element click intercepted: Element <input type="radio" id="impressiveRadio" name="like" class="custom-control-input"> is not clickable at point (345, 376). Other element would receive the click: <label class="custom-control-label" for="impressiveRadio">...</label>

Reply:
You are still using the input and not the label as I've showed you yesterday. Please make this adjustment. Thanks!